src/docs/Glossary.adoc

```diff
@@ -34,7 +34,7 @@ Scheduler Web Interface:: ProActive component that provides a web interface to t
 Workflow Studio:: ProActive component that provides a web interface for designing <<_glossary_workflow,*Workflows*>>.
 
 [[_glossary_automation_dashboard]]
-Automation Dashboard:: Centralized interface for the following web portals: <<_glossary_workflow_automation_portal,*Workflow Automation*>>, <<_glossary_catalog_portal,*Catalog*>>, <<_glossary_analytics_portal,*Analytics*>>, <<_glossary_job_planner_portal,*Job Planner*>>, <<_glossary_service_automation_portal,*Service Automation*>>, <<_glossary_event_orchestration_portal,*Event Orchestration*>> and <<_glossary_notification_portal,*Notifications*>>.
+Automation Dashboard:: Centralized interface for the following web portals: <<_glossary_workflow_automation_portal,*Workflow Execution*>>, <<_glossary_catalog_portal,*Catalog*>>, <<_glossary_analytics_portal,*Analytics*>>, <<_glossary_job_planner_portal,*Job Planner*>>, <<_glossary_service_automation_portal,*Service Automation*>>, <<_glossary_event_orchestration_portal,*Event Orchestration*>> and <<_glossary_notification_portal,*Notifications*>>.
```

Comment:
@oanaBiancaSchiopu glossary_workflow_automation_portal should not be changed ?

Reply:
maybe see with @amouhoub if he already planned to remove WA doc

Reply:
@amouhoub has removed WA